.github/issue_template.md

```diff
@@ -0,0 +1,35 @@
+- Is this an issue with SCAP Workbench?
+    - If so, report it here: https://github.com/OpenSCAP/scap-workbench/issues
+- Is this an issue with SCAP Security Guide (i.e., related to the content of scans, not the scanner proper)?
+    - If so, report it here: https://github.com/OpenSCAP/scap-security-guide/issues
+- Is this an issue during the OS installation process?
```

Comment:
Would you prefer this says RHEL / etc? It isn't default on any OS so this should be a rare enough occurrence and depending on the issue, it might actually be an `openscap` issue, but I'm not sure what we want the desired behavior to be.

Reply:
IMO it's fine as is.